src/frontends/sql/testing/merge_operation_view.h

```diff
@@ -32,17 +33,30 @@ class MergeOperationView {
   MergeOperationView(const ir::Operation &merge_operation)
       : merge_operation_(&merge_operation) {
     CHECK(merge_operation.op().name() == DecoderContext::kSqlMergeOpName);
-    CHECK(merge_operation.attributes().empty());
+    CHECK(merge_operation.attributes().count(
+              DecoderContext::kMergeOpControlStartIndex) > 0);
```

Comment:
Might also want to add a check that it is less than the total number of inputs.

Reply:
done. needed to jump through too many hoops!